put-a-jetpack-on-it/slides.md

```diff
@@ -0,0 +1,305 @@
+# Putting a Jetpack on your legacy codebase
+
+## Kurt Nelson
+### Android Platform at Pinterest
+
+Hey everyone, Kurt from the Pinterest Android Platform (or Android Experience) team.
+
+---
+What do I work on?
+
+# Platform Teams
+	- Mobile-wide issues
+	- Build, framework, testing
+	- Android, iOS, Web
+
+At many of the bigger tech companies with longstanding big apps, we spin up platform and infrastructure teams within engineering to concentrate on mobile-wide issues. Sister teams of ours for example are the mobile builds team and the test tooling team. There are also equivalents for iOS, Web, etc.
+
+---
+# Forward-looking big picture
+
+### Ensuring the stability, security and performance going forward
+
+### Library choices and upgrades
+
+One of our roles is to look at both the big picture and the future at the same time, hopefully heading external changes in best practices off enough that feature teams can write features and not have to worry about things like long term stability, performance, and library upgrades.
+
+---
+# We experiment early with new technologies
+
+This means we are also in charge of trying out new libraries, best practices, and patterns to see if they belong in our app.
+
+---
+# We pay attention to what is going on in the Android world
+	hi room.
+
+Platform teams are often in the room early, even possibly getting first access to libraries to play with.
+
+---
+# Handling Android Changes
+
+Of course, when Google makes large changes, we don’t as much have a choice of if we will use it but of when we will pick up the changes. 
+
+---
+### Handling Android Changes
+# Kotlin
+
+Kotlin was a great example of this- At Pinterest, we were pretty early to stamp it as generally available in our codebase. However over at Uber, it was causing build performance regressions annoying enough that it was just marked as GA recently in their code base.
+
+---
+### Handling Android Changes
+# Jetpack Compose
+
+On the flip side, we still have not given Jetpack Compose the platform team blessing as we see meaningful regressions in the performance of our home feed when we run A/B tests. 
```

Comment:
it mean a bit cleaner if it was:
"the platform team  has not given Jetpack Compose the blessing"